packages/fx-core/src/common/samples-config.json

```diff
@@ -20,6 +20,13 @@
             "time": "5min to run",
             "configuration": "F5 ready"
         },
+        {
+            "id": "graph-toolkit-contact-exporter",
+            "title": "Contact Exporter using Graph Toolkit",
+            "shortDescription": "Contact Exporter app of Microsoft Teams Tab app.",
+            "fullDescription": "Contact Exporter app can export your teams contact info to Excel files, and it is using Microsoft Graph Toolkit as UI component. This app only has a Tab frontend.",
```

Comment:
I added estimation time and configuration info for all samples, you need to update related info to align with all samples.  My pr: https://github.com/OfficeDev/TeamsFx/pull/3055

Reply:
Updated, thanks!